docs/en/Running-Alluxio-On-Docker.md

```diff
@@ -141,11 +141,33 @@ $ cd opt/alluxio
 $ bin/alluxio runTests
 ```
 
-### Sharing ramdisk with clients
+### Read/Write to the local worker 
 
-Running the worker with `-v /mnt/ramdisk:/mnt/ramdisk` shares the ramdisk between the worker
-and host. To make this ramdisk available to clients in other containers running on the same host,
-those containers should also be run with `-v /mnt/ramdisk:/mnt/ramdisk`.
+There are two options to read/write to the local worker efficiently. The option that shares ramdisk
+with the client requires the memory limit of the client container to be greater than the amount of
+data it writes to the local ramdisk. The domain socket approach does not have this
+limitation but it consumes more CPU on the client container and the worker container.
+
+#### Share ramdisk with clients
+Running the worker with `-v /mnt/ramdisk:/opt/ramdisk` shares the ramdisk between the worker 
+(`/opt/ramdisk`) and host (`/mnt/ramdisk`). To make this ramdisk available to clients in 
+other containers under `/opt/ramdisk` running on the same host, those containers should also be 
+run with `-v /mnt/ramdisk:/opt/ramdisk`.
+
+#### Share domain socket with clients
+
+From the host machine
+```bash
+$ mkdir /tmp/domain
+$ chmod a+w /tmp/domain
+$ touch /tmp/domain/d
+$ chmod a+w /tmp/domain/d
+```
+When starting worker and clients, run the docker container with `-v /tmp/domain:/opt/domain`
+to share `/tmp/domain` directory on the host machine with the worker and clients under 
+`/opt/domain`. And set the site property `alluxio.worker.data.server.domain.socket.address` 
+in the worker by passing `-e ALLUXIO_WORKER_DATA_SERVER_DOMAIN_SOCKET_ADDRESS=/opt/domain/d` 
+when running the container.
```

Comment:
`-e ALLUXIO_WORKER_DATA_SERVER_DOMAIN_SOCKET_ADDRESS=/opt/domain/d`  does that ? Or you think we need further clarification?